### DISCUSSION TITLE
除了Desktop应用，我自定义的应用或者Steam大屏启动都报错Error 0

### DISCUSSION THREAD

**Discussion by ChenZion** [ClassicOldSong/Apollo#19] [Q&A] [answered]
笔记本电脑：630核显+GTX1060
操作系统版本：Window 10
登录用户：Administrator
安装Apollo路径：C:\Program Files\Apollo

Moonlight报错信息：
![image](https://github.com/user-attachments/assets/401a7699-545a-419f-89c6-a3a7de291497)

Apollo异常日志：
![image](https://github.com/user-attachments/assets/63267696-7e58-4196-9a94-00618f2d6d6b)

根据异常日志提示权限问题。但是我启动Apollo已经是用管理员身份运行，也在属性勾选已管理员身份运行，并且给文件夹权限配置所有用户完全控制权限。

请问该如何解决该问题？ :(

**Comment by ClassicOldSong** [comment]
你是在开始菜单里启动Apollo的吗？仅用管理员权限运行其实依旧有限制，需要服务启动，开始菜单里打开的话会自动以服务启动Apollo。

而且APP不要勾选使用管理员权限启动，或者你可能没正常安装Steam。

**Reply by ChenZion** [reply]
您好作者大大，我是在桌面的Apollo快捷方式图标设置的。我尝试过APP有无勾选“使用管理员权限启动”都会有这种问题

**Reply by ClassicOldSong** [reply]
你试试看添加别的程序呢？不要用Steam ID启动试试看？

**Reply by ChenZion** [reply]
我尝试添加一个E盘的游戏
![image](https://github.com/user-attachments/assets/415a7f9b-cbbf-4c7e-8963-a0c55322f135)
这是我APP的配置
![image](https://github.com/user-attachments/assets/7dd573c5-a145-4398-bbdd-552deecdee70)
也是同样问题

**Reply by ClassicOldSong** [reply]
杀毒软件关一下看看，可能是被拦截启动了

**Reply by ClassicOldSong** [reply]
问题解决了吗？

**Reply by ChenZion** [reply]
抱歉昨晚太困了。今天我尝试已经把防火墙和杀毒软件都关掉，也是不行。昨天使用MacBook连接，今天用Windows电脑连接，也是会爆同样问题。
![image](https://github.com/user-attachments/assets/9984f19d-ed4e-4b9e-8df5-07bcb67a3c8f)

**Reply by ChenZion** [reply]
您好作者，目前我已经解决了该问题。但是不知道如何触发这个异常。
我操作过的步骤：
1. 给Apollo文件夹和子文件夹，在属性——安全——给Administrator用户和everyone用户配置“完全控制”权限；
2. 本地安全策略——本地策略——安全选项——【用户账户控制：以管理员批准模式运行所有管理员】 修改为“已禁用”、【用户账户控制：用于内置管理员账户的管理员批准模式】修改为“已禁用”，重启电脑；
3. 在Apollo WebUI控制页面，设置——Network——【IP地址族：IPv4+IPv6 】 改为：仅IPv4
4. 在Apollo WebUI控制页面，设置——Network——【公网加密模式：禁用】   改为 “为支持的客户端启用（默认）”
5. 重启服务后连接恢复正常；

目前触发这个问题的条件不知道是什么，但是在以上操作完成后，执行完3/4/5步骤，Moonlight客户端就可以正常启动APP了。：)

**Reply by ClassicOldSong** [reply, marked as answer]
查了一下，具体原因应该是System权限并不一定能正常访问用户可访问的文件，可能碰巧你的Steam和其他游戏System权限访问不了。。。

参考：
https://docs.lizardbyte.dev/projects/sunshine/en/latest/troubleshooting/windows.html

**Reply by ClassicOldSong** [reply]
你的操作中实际有效的应该只有第二步

**Reply by ChenZion** [reply]
那应该是后面做的操作只是触发了重启服务刷新了权限，实际修改有效是第二步骤。目前整体运行稳定了。感谢作者大大
